helix-term/src/commands.rs

```diff
@@ -431,7 +431,7 @@ impl MappableCommand {
         decrement, "Decrement item under cursor",
         record_macro, "Record macro",
         replay_macro, "Replay macro",
-        command_palette, "Open command palette",
+        command_palette, "Open command pallete",
```

Comment:
Shouldn't this be palette like the command?

Reply:
https://www.merriam-webster.com/dictionary/palette is the correct spelling